_config.yml

```diff
@@ -52,4 +52,18 @@ footer_links:
     text: Facebook
     newTab: true
 
+collections_dir: city_local/berlin
+future: true  
```

Comment:
ah, very cool

Reply:
I can imagine changing events page, to show upcoming events, versus past events.